### ISSUE TITLE
Web version doesn't keep its aspect ratio, causing it to look weird and elements falling off.

### ISSUE THREAD

**Issue by BlackDragonBE** [BlackDragonBE/MarkdownToRWHTMLUnity#7]
I should be able to force the canvas to retain its aspect ratio in html and disable the fullscreen option.

**Comment by BlackDragonBE** [comment]
```js
function resizeCanvas() {
    canvas = document.getElementById("c");
    gl = canvas.getContext("webgl");
    var width = canvas.clientWidth;
    var height = canvas.clientHeight;
    if (canvas.width != width || canvas.height != height) {
        canvas.width = width;
        canvas.height = height;

        gl.viewport(0, 0, gl.canvas.width, gl.canvas.height);
    }
}
```

**Comment by BlackDragonBE** [comment]
Fixed in 1.10
